recipe/meta.yaml

```diff
@@ -26,6 +27,7 @@ requirements:
       - pip
       - numpy
       - pyparsing
+      - epicscorelibs
```

Comment:
Neat, thanks for putting efforts into packaging it via https://github.com/conda-forge/epicscorelibs-feedstock! I didn't know about this package before.

Do you mind to sort the dependencies alphabetically (besides `python`)? This way it's easier to locate the dependency in the potentially long list, as well as to avoid duplicates. We try to do it for our new recipes in https://github.com/nsls-ii-forge.

Reply:
Addressed with latest commit

Reply:
Appreciate it!